OpenTESArena/src/Utilities/Debug.cpp

```diff
@@ -49,25 +50,30 @@ void Debug::write(Debug::MessageType type, const std::string &filePath,
 
 void Debug::mention(const char *__file__, int lineNumber, const std::string &message)
 {
-	Debug::write(Debug::MessageType::Info, Debug::getShorterPath(__file__), 
+	Debug::write(Debug::MessageType::Info, Debug::getShorterPath(__file__),
 		lineNumber, message);
 }
 
 void Debug::warning(const char *__file__, int lineNumber, const std::string &message)
 {
-	Debug::write(Debug::MessageType::Warning, Debug::getShorterPath(__file__), 
+	Debug::write(Debug::MessageType::Warning, Debug::getShorterPath(__file__),
 		lineNumber, message);
 }
 
 void Debug::crash(const char *__file__, int lineNumber, const std::string &message)
 {
 	Debug::write(Debug::MessageType::Error, Debug::getShorterPath(__file__),
 		lineNumber, message);
-	std::getchar();
+    const std::string platformName(SDL_GetPlatform());
+    if (platformName == "Mac OS X") {
+        SDL_ShowSimpleMessageBox(SDL_MESSAGEBOX_ERROR, "Error", message.c_str(), NULL);
```

Comment:
In C++11, the desired way to indicate a null pointer is with `nullptr`, not `NULL`. The rest looks good, though.